src/components/annotations3d/attributes.js

```diff
@@ -75,13 +75,13 @@ module.exports = {
     arrowwidth: annAtts.arrowwidth,
     standoff: annAtts.standoff,
     hovertext: annAtts.hovertext,
-    hoverlabel: annAtts.hoverlabel
+    hoverlabel: annAtts.hoverlabel,
+    captureevents: annAtts.captureevents
```

Comment:
After 1d5606a, [this](https://github.com/plotly/plotly.js/blob/78b82b85206bde93c7117706278ac698b480139d/test/jasmine/tests/annotations_test.js#L1202) and [that](https://github.com/plotly/plotly.js/blob/78b82b85206bde93c7117706278ac698b480139d/test/jasmine/tests/annotations_test.js#L1226) assertions are failing, because the event data now contains `subplotId: false` for cartesian/paper-ref annotations.

@alexcjohnson would you prefer leaving `subplotId` out of the cartesian/paper-ref annotation event data or keep it as `subplotId: false`?